flake.nix

```diff
@@ -6,6 +6,9 @@
 
     nixpkgs.follows = "haskell-nix/nixpkgs-2105";
```

Comment:
Yikes that's an old version of `nixpkgs` (I saw the TODO below). Do you think you could try switching to a more recent release in this PR?

Reply:
updated from staging